deploy/ocs-operator/manifests/blackbox-exporter-role-binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
@yati1998 this file is getting add here on `make gen-latest-csv` this should have been part of https://github.com/red-hat-storage/ocs-operator/pull/3565/ right?